website/content/tutorials/1-getting-started/_index.md

```diff
@@ -0,0 +1,954 @@
+}
+{{< /highlight >}}
+{{% /tab %}}
+{{% tab %}}
+{{< highlight xml >}}
+<?xml version="1.0" encoding="UTF-8"?>
+<computer xmlns="http://example.com/ns/computer" id="awesomepc1">
+  <vendor id="vendor1">
+    <name>AwesomeComp Incorportated</name>
+    <address>1000 K Street NW Washington, DC 20001</address>
+    <website>https://example.com/awesomecomp/awesomepc1/details</website>
+  </vendor>
+  <motherboard>
+    <product-name>ISA Corp Magestic Model M-Ultra Motherboard</product-name>
+    <type>atx</type>
+    <cpu>
+      <product-name>ISA Corp Superchip Model 1 4-core Processor</product-name>
+      <architecture>x86-64</architecture>
+      <speed>4.7 gigahertz</speed>
+    </cpu>
+    <ata-socket>
+      <product-name>ISA Corp SuperSATA Model 2 Storage Socket</product-name>
+      <type>sata</type>
+    </ata-socket>
+    <memory>
+      <product-name>Massive Memory Corp Model 3 DDR4-3200 8GB (Module 1)</product-name>
+      <byte-size>8589934592</byte-size>
+    </memory>
+    <memory>
+      <product-name>Massive Memory Corp Model 3 DDR4-3200 8GB (Module 2)</product-name>
+      <byte-size>8589934592</byte-size>
+    </memory>
+  </motherboard>
+</computer>
+{{< /highlight >}}
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml >}}
+---
+computer:
+  id: awesomepc1
+  vendor:
+    id: vendor1
+    name: AwesomeComp Incorportated
+    address: 1000 K Street NW Washington, DC 20001
+    website: https://example.com/awesomecomp/awesomepc1/details
+  motherboard:
+    product-name: ISA Corp Magestic Model M-Ultra Motherboard
+    type: atx
+    cpu:
+      product-name: ISA Corp Superchip Model 1 4.7 GHz 4-core Processor
+      architecture: x86-64
+      speed: 4.7 gigahertz
+    ata-socket:
+      product-name: ISA Corp SuperATA Model 2 Socket for SATA Drive
+      type: sata
+    memory-modules:
+    - product-name: Massive Memory Corp Model 3 8GB DDR4-3200 (Module 1)
+      size: 8589934592
+    - product-name: Massive Memory Corp Model 3 8GB DDR4-3200 (Module 2)
+      size: 8589934592
+{{< /highlight >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+With the expressive power of assemblies, flags, and fields, we can specify complex managed objects and control the structure of the intended information model in the resulting data formats.
+
+Our Metaschema-enabled tools can parse and generate the different data formats. We specify flags on the `computer` assembly, but all else we define in the `model` of the `assembly`. And within that model, we can define the motherboard `assembly` inline with its own `flag` and `model`. XML has different syntax and semantics from JSON and YAML that would lead to structural limitations we would need to work around ourselves by manually designing separate data formats without Metaschema. With Metaschema, we can use `group-as` declarations  in our definition, and Metaschema-enabled tools will generate and parse instances for us as instances of the same, despite these limitations, in different data formats automatically.
```

Comment:
Suggest replacing from "XML has different syntax and semantics..." with (something like):

> These abstract definitions, along with information we provide with them such as names of groups, enables a Metaschema-aware engine to sort out and distinguish the data points as we wish them to appear differently, in different syntaxes. So a JSON schema can describe a JSON format that is idiomatic in JSON, for storing or transmitting a data set, while an XSD can describe a *different* format idiomatic in XML, for the same data set. Similarly, a processor configured with a Metaschema definition can parse any of these syntaxes into a model with a defined (deterministic) mapping to the abstract (Metaschema) model -- rendering the data independent of the notation used to represent it, and rewritable in any notation or format supported by the processor.

Reply:
I was actually thinking of turning this into a footnote prior to this review comment and seems I forgot, I need to come back to this later in the review cleanup.

Reply:
Actually, re-reading this, I changed my mind I will work this in.

Reply:
```suggestion
Our Metaschema-enabled tools can parse and generate the different data formats. We specify flags on the `computer` assembly, but all else we define in the `model` of the `assembly`. And within that model, we can define the motherboard `assembly` inline with its own `flag` and `model`. These abstract definitions, along with information we provide with them such as names of groups, enables a Metaschema-enabled tool to sort out and distinguish the data points as we wish them to appear differently in a different syntax. A JSON schema can describe a JSON format that is idiomatic in JSON, while an XML Schema can do the same in XML with the same Metaschema model. As this example demonstrates, Metaschema allows developers to render the data independent of the notation used to represent it, and convert into any other notation their tools to support.
```